src/components/cvaa-overlay/main-captions_window.js

```diff
@@ -86,20 +86,23 @@ class MainCaptionsWindow extends Component {
             changeCaptionsStyle={props.changeCaptionsStyle}
             captionsStyle={this.captionsStyleDefault}
             player={props.player}
+            buttonNumber={'1'}
```

Comment:
sampleNumber instead of buttonNumber is more descriptive

Reply:
done